### PR TITLE
Fix some broken links

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: appendices/VK_NV_compute_shader_derivatives.txt [hunk 1]
```diff
@@ -24,12 +24,12 @@ compute shaders to use built-ins that evaluate compute derivatives
...
-invocation ID of the form (2m+{0,1}, 2n+{0,1}).
-The code:DerivativeGroupLinearNV execution mode assembles shader invocations
-into 2x2 groups, where each group has local invocation index values of the
-form 4m+{0,1,2,3}.
+The code:ComputeDerivativeGroupQuadsNV execution mode assembles shader
+invocations into 2x2 groups, where each group has x and y coordinates of the
+local invocation ID of the form (2m+{0,1}, 2n+{0,1}).
+The code:ComputeDerivativeGroupLinearNV execution mode assembles shader
+invocations into 2x2 groups, where each group has local invocation index
+values of the form 4m+{0,1,2,3}.
```
Comment (jeffbolznv) [inline appendices/VK_NV_compute_shader_derivatives.txt:32]: Isn't the execution mode name correct without `Compute` (and it's the capability that has `Compute` on it?

**Comment by jekstrand** [comment]
Intel bit looks fine.

File: appendices/VK_NV_compute_shader_derivatives.txt (hunk 1 above)
Comment (krOoze) [reply appendices/VK_NV_compute_shader_derivatives.txt:32]: Yea, I think so. I was fixing a link to a derivatives chapter that does not exist anymore. I don't even remember changing it. Damn heat...

Comment (krOoze) [reply appendices/VK_NV_compute_shader_derivatives.txt]: should be gone now

**Pull request merged**